s3tests_boto3/functional/utils.py

```diff
@@ -1,7 +1,9 @@
+import allure
 import random
 import string
 
 
+@allure.step("Assert Raises")
```

Comment:
Is it a proper step, btw?

Reply:
In my opinion, it is. As for tests, checking negative scenarios assertion with expected error is just a step. Like in [test_object_copy_bucket_not_found](https://github.com/nspcc-dev/s3-tests/blob/a35437900562ae3b17797fe82ca7c2c1840665fe/s3tests_boto3/functional/test_s3.py#L6768C56-L6768C56)

And it looks like this in test report:
<img width="652" alt="image" src="https://github.com/nspcc-dev/s3-tests/assets/13661886/45a3d3ff-93ab-45e5-8c7d-6efdebedfbd0">